pages/profile/academics/index.js

```diff
@@ -1,10 +1,16 @@
 import Head from "next/head";
 import React from "react";
+//styles
 import { Content, Header } from "../../../components/layout/layout";
 import Segment from "../../../components/layout/segment/segment";
+
+//graphing
 import DoughnutChart from "../../../components/charts/doughnut";
 import Scatterplot from "../../../components/charts/scatterplot";
-import BarGraph from "../../../components/charts/bar";
+import Bar from "../../../components/charts/bar";
+
+//writeup
+import Text from "../../../text1A";
```

Comment:
We’re moving the write ups to firebase, we can remove the text1A file & references